index.html

```diff
@@ -48,347 +44,346 @@
   <body
     x-data="gameState()"
     x-init="initialLoad()"
+    :class="`theme-${settings.theme}`"
     @keyup.document="keyPressed"
     @hashchange.window="page = location.hash"
   >
-    <div class="theme-container" :class="`theme-${settings.theme}`">
```

Comment:
hide whitespace to see that I only removed theme-container - wasn't needed after all